classes/class-wc-connect-shipping-label.php

```diff
@@ -111,22 +117,38 @@ protected function get_selected_packages(WC_Order $order ) {
 		}
 
 		protected function get_all_packages() {
-			$packages = $this->settings_store->get_packages();
-
-			if ( ! $packages ) {
-				return new stdClass();
-			}
+			$custom_packages = $this->settings_store->get_packages();
 
 			$formatted_packages = array();
 
-			foreach( $packages as $package ) {
+			foreach( $custom_packages as $package ) {
 				$package_id = $package[ 'name' ];
 				$formatted_packages[ $package_id ] = $package;
 			}
 
+			$flat_rate_schema = $this->service_schemas_store->get_predefined_packages_schema_for_service( 'usps' );
```

Comment:
If we filter on enabled services only, could we avoid hardcoding `usps` here and make this support predefined packages for other carriers, like FedEx One Rate boxes?